tests/integration/esgf/test_search_download.py

```diff
@@ -115,14 +128,40 @@ def test_mock_search(variable, mocker):
 
     files = find_files(**variable)
 
-    with open(data_path / 'expected.yml') as file:
-        expected_files = yaml.safe_load(file)[json_file]
+    expected_results_file = data_path / 'expected.yml'
+    if expected_results_file.exists():
+        with expected_results_file.open(encoding='utf-8') as file:
+            expected_results = yaml.safe_load(file)
+    else:
+        expected_results = {}
+    print(expected_results)
```

Comment:
```suggestion
```

Reply:
Why would you recommend removing the print statement? I found it quite helpful for debugging when I was writing the test.